modules/workbench/cloud9/modulestack.yaml

```diff
@@ -0,0 +1,26 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This stack deploys a Module specific IAM permissions
+
+Parameters:
+  # DeploymentName:
+  #   Type: String
+  #   Description: The name of the deployment
+  # ModuleName:
+  #   Type: String
+  #   Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action:
+              - "ec2:ModifyVolume"
+            Resource: "*"
```

Comment:
Resource is too wide open...gives the deployment role the ability to change any ec2 volume on any instance